package.json

```diff
@@ -30,8 +31,8 @@
     "eslint-plugin-es": "^4",
     "eslint-plugin-filenames": "^1",
     "eslint-plugin-import": "^2",
-    "eslint-plugin-prettier": "^3 || ^4",
```

Comment:
Do we want to allow `^3 || ^4 || ^5`? I don't remember why we have `^3 || ^4` here in the first place. Arguably we should be specifying a major version we expect for our eslint-plugin? If we don't have a prettier requirement can we remove it altogether?